unified-prototype/unified-plugin/plugin-android/src/main/java/org/gradle/api/experimental/android/AbstractAndroidSoftwarePlugin.java

```diff
@@ -0,0 +1,179 @@
+package org.gradle.api.experimental.android;
+
+import com.android.build.api.dsl.*;
```

Comment:
Can we avoid these star imports?

Doesnt have to be done now but lets check-in some Intellij style rules to this project